Makefile

```diff
@@ -1,21 +1,39 @@
-start:
-	test -f .env || make local-config
-	test -d Packages || swift package edit PointFree
-	docker-compose up --build
+oss: .env Sources/pointfreeco/Transcripts/
+	swift run
+
+.env:
+	test -f .env \
+		|| cp .env.example .env
+
+Sources/pointfreeco/Transcripts/:
+	test -d Sources/pointfreeco/Transcripts/ \
+		|| cp -r Transcripts.example/ Sources/pointfreeco/Transcripts/
+
+# bootstrap
+
+install-cmark:
+	apt-get -y install cmake
+	git clone https://github.com/commonmark/cmark
+	make -C cmark INSTALL_PREFIX=/usr
+	make -C cmark install
+
+db:
+	createuser --superuser pointfreeco || true
+	createdb --owner pointfreeco pointfreeco_development || true
+	createdb --owner pointfreeco pointfreeco_test || true
+
+xcodeproj:
+	swift package generate-xcodeproj
+	xed .
 
-transcripts:
-	git submodule update --init --recursive
-	ln -fn Transcripts/*.swift Sources/pointfreeco/Transcripts/
-	git update-index --assume-unchanged Sources/pointfreeco/Transcripts/AllEpisodes.swift
-	echo Sources/pointfreeco/Transcripts/ > .git/info/exclude
-	make xcodeproj
+# private
 
-untranscripts:
-	echo > .git/info/exclude
-	git update-index --no-assume-unchanged Sources/pointfreeco/Transcripts/AllEpisodes.swift
-	git checkout -- Sources/pointfreeco/Transcripts/
-	git clean --force Sources/pointfreeco/Transcripts/
-	make xcodeproj
+start:
```

Comment:
We can make our private bootstrap process better in the future, but for now I'm prioritizing OSS.